css/style.css

```diff
@@ -1,89 +1,130 @@
+/*** Colors ***/
+
+:root {
+  --link: #0c60ff;
+  --link-hover: #0c60ffaa;
+}
+
+/*** Core Elements ***/
+
 body {
   font-family: "Helvetica", sans-serif;
-  font-size: 1.15em;
+  font-weight: 400;
+  line-height: 1.5;
+  padding: 0 20px;
   margin: 0 auto;
-  padding: 0;
-  width: 100%;
+  margin-top: env(safe-area-inset-top, 0);
+  margin-bottom: env(safe-area-inset-bottom, 0);
+  max-width: 960px; /* TODO: change to ch based */
```

Comment:
I'm leaving this TODO for now, because I would rather it be included in a focused change that rethinks the width breakpoints.